wiki/Advancement-Type:-Hit-Points.md

```diff
@@ -0,0 +1,46 @@
+![Up to date as of 2.0.3](https://img.shields.io/static/v1?label=dnd5e&message=1.6.3&color=informational)
+
+The Hit Points advancement type keeps track of hit point choices for each level of a class. This advancement can only be added to classes and each class can only have a single advancement of this type.
+
+## Configuration
+
+<img width="400" alt="Hit Points - Configuration" src="https://user-images.githubusercontent.com/19979839/195448359-c3274376-90e3-4bad-850a-02958039a026.png">
```

Comment:
These should also work with markdown formatting, no?
```suggestion
![Hit Points - Configuration](https://user-images.githubusercontent.com/19979839/195448359-c3274376-90e3-4bad-850a-02958039a026.png)
```
Or are we using HTML for the `width="400"` bit?

Reply:
I just reverted this back to what it was in the wiki originally, I pulled the wiki down to this repo https://github.com/MaxPat931/dnd5e/wiki/ 
But yea I find these ones a little odd that they have the extra formatting to make them smaller, so I'll change them over to the standard markdown format